UtxoSelector component

ronaldsg20 opened the pull request:
I've created de Utxo selector modal in order to provide users the ability
		to ensure wich utxos will be availables on the powPegApp

alexjavabraz reviewed (approved):
LGTM

annipi commented:
<img width="1290" alt="Screenshot 2025-05-13 at 5 37 14 PM" src="https://github.com/user-attachments/assets/3e741292-a495-4df5-ab6f-3cad40349ea9" />

I think the distance between the select utxos button and the theme button is too long

annipi commented:
with leather I'm not able to see my balance, after selectiong the "select utxos" button 
<img width="1265" alt="Screenshot 2025-05-13 at 5 44 46 PM" src="https://github.com/user-attachments/assets/ee8b2e82-0291-4bc2-a6ff-c7df88c9a18a" />

Also, should this button of "select utxos" be instead an option on the dropdown from the top bar?

annipi commented:
Enkrypt is also failing in the same way

annipi commented:
I think the top titles are mixed for `Segwit` and `Native Segwit`, sending video internally to show the issue

annipi commented on src/common/components/layouts/Top.vue:
```diff
@@ -28,6 +28,8 @@
         <span class="slider"></span>
       </label>
     </div>
+    <v-btn @click="displayUtxoSelector">select Utxos</v-btn>
```
Shouldn't this button be better placed inside the dropdown as an extra option?

annipi commented on src/pegin/store/PeginTx/actions.ts:
```diff
@@ -214,4 +214,9 @@ export const actions: ActionTree<PegInTxState, RootState> = {
   [constants.PEGIN_TX_SET_CURRENT_VIEW]: ({ commit }, view: string): void => {
     commit(constants.PEGIN_TX_SET_VIEW, view);
   },
+  // eslint-disable-next-line no-empty-pattern
```
I don't think this line is necessary

annipi reviewed (changes requested):
I left some comments that need further revision

ronaldsg20 commented:
![image](https://github.com/user-attachments/assets/efb04f1e-983f-40ca-93e6-eabb52137ae6)
Fixed the button location (on an icon)
Fixed the account types mix, it solves also the Leather problem

annipi reviewed (approved):
LGTM!
